app/main/views/briefs.py

```diff
@@ -360,15 +360,21 @@ def view_response_result(brief_id):
         return redirect(url_for(".brief_response", brief_id=brief_id))
     elif brief_response[0].get('essentialRequirementsMet') or all(brief_response[0]['essentialRequirements']):
         result_state = 'submitted_ok'
+        flash('submitted_ok', 'success')
```

Comment:
I've spoken to @ralph-hawkins and he says the flash message should be every time you visit the page after a successful submission. He also suggested rationalising how the banners (and h1s) work for both flows. Notes have been added to [the story](https://www.pivotaltracker.com:/story/show/130019135/comments/161850371) which list the changes needed for this.